src/python/pants/option/custom_types.py

```diff
@@ -214,11 +223,13 @@ def create(cls, value) -> "ListValueComponent":
             appends = value
         elif value.startswith("[") or value.startswith("("):
             action = cls.REPLACE
-            appends = _convert(value, (list, tuple))
+            appends = _convert_list(value, member_type, is_enum)
         elif value.startswith("+[") or value.startswith("+("):
-            appends = _convert(value[1:], (list, tuple))
+            appends = _convert_list(value[1:], member_type, is_enum)
         elif value.startswith("-[") or value.startswith("-("):
-            filters = _convert(value[1:], (list, tuple))
+            filters = _convert_list(value[1:], member_type, is_enum)
+        elif is_enum and isinstance(value, str):
+            appends = _convert_list(value.split(","), member_type, True)
```

Comment:
I think this is the cause of the bad change mentioned below in the tests. We don't want to split based on commas.